rotkehlchen/accounting/structures/base.py

```diff
@@ -119,6 +120,17 @@ def __init__(
         self.notes = notes
         self.identifier = identifier
 
+        # Check that the received event type and subtype is a valid combination
+        if __debug__:  # noqa: SIM102
```

Comment:
What is the SIM102 for?

Reply:
this is to avoid ruff consolidating both of the if checks in one. Afaik if is not just `if __debug__:` it won't be removed on the production binary

Reply:
Please resolve if looks good.